src/densities.jl

```diff
@@ -79,3 +79,23 @@ be one coefficient matrix per k-Point.
     count = sum(length(basis.ksymops[ik]) for ik in 1:length(basis.kpoints))
     from_fourier(basis, sum(ρaccus) / count)
 end
+
+"""
+Computes the *probability* (not charge) current, ∑ fn Im(ϕn* ∇ϕn)
+"""
+function compute_current(basis::PlaneWaveBasis, ψ::AbstractVector,
+                         occupation::AbstractVector)
+    @assert all(symop -> length(symop) == 1, basis.ksymops) == 1  # TODO lift this
```

Comment:
I think we should have a function for that that checks whether symmetry is on for a model or the basis.

Reply:
I'm fine with this, it's not like we do it a lot, and it's all when we're too lazy to figure out the right expressions

Reply:
ok